src/components/Widgets/WalletButton.jsx

```diff
@@ -0,0 +1,106 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+
+export default class WalletButton extends Component {
+  static displayName = 'WalletButton'
+
+  static propTypes = {
+    /** Display text to be displayed as button symbol */
+    buttonSymbol: PropTypes.string,
+    /** Display text describing button action */
+    innerText: PropTypes.string,
+    /** Callback to be executed onClick */
+    onClick: PropTypes.func
+  }
+
+  static defaultProps = {
+    buttonSymbol: '+',
+    innerText: ''
+  }
+
+  handleClick = e => {
+    const { onClick } = this.props
+    if (onClick) {
+      onClick(e)
+    }
+  }
+
+  render() {
+    const { buttonSymbol, innerText } = this.props
+    return (
+      <StyledButton onClick={this.handleClick}>
```

Comment:
Buttons need some additional configurability (e.g. if within a form, to be able to set `type="submit"`). Please include `{...this.props}` to pass all props through.

Reply:
done.